eslint.config.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @author 唯然<weiran.zsd@outlook.com>
+ */
+"use strict"
+
+const js = require("@eslint/js")
+const { FlatCompat } = require("@eslint/eslintrc")
+const globals = require("globals")
+const nodeRecommended = require("eslint-plugin-n/configs/recommended-script")
+
+const compat = new FlatCompat({
+    baseDirectory: __dirname,
+    recommendedConfig: js.configs.recommended,
+})
+
+module.exports = [
```

Comment:
The new config system doesn't support `.eslintignore` files, so if we want to ignore something we should add it here.

```js
{
    ignores: [
        ".nyc_output/",
        "coverage/",
        "docs/",
        "lib/converted-esm/"
    ]
}
```

https://eslint.org/docs/latest/use/configure/configuration-files-new#globally-ignoring-files-with-ignores

Reply:
addressed in f2b5712c10b2812fa924f21bc10f905509814a03